src/SConscript

```diff
@@ -18,14 +24,25 @@ def scons():
     env.AppendUnique(CPPPATH=[Dir('include/daos').srcnode()])
     env.Install(os.path.join('$PREFIX', 'include/daos'), 'include/daos/tse.h')
 
+    env.AppendUnique(LIBPATH=[Dir('gurt')])
```

Comment:
This should probably be in gurt/SConscript and be LIBPATH=[Dir('.')] instead before cloning the environment

Reply:
Done, thank you.